libraries/streaming-e2e-tests/react-app/public/index.html

```diff
@@ -9,6 +9,15 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <meta http-equiv="Content-Security-Policy"
+        content="default-src 'none'; 
+            base-uri 'none'; 
+            connect-src wss://*.azurewebsites.net https://*.azurewebsites.net; 
```

Comment:
```suggestion
            connect-src blob: wss://%REACT_APP_BOT_HOSTNAME%.azurewebsites.net https://%REACT_APP_BOT_HOSTNAME%.azurewebsites.net;
```

Reply:
Yeah! Figured out the env variable, which is defined in Az Pipelines was (once again) defined in wrong job (ugh). This works now :) 

CSP updated